contrib/signet/generate.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+# Copyright (c) 2010 ArtForz -- public domain half-a-node
+# Copyright (c) 2012 Jeff Garzik
+# Copyright (c) 2010-2020 The Bitcoin Core developers
```

Comment:
```suggestion
# Copyright (c) 2020 The Bitcoin Core developers
```

:eyes:

Reply:
Hmm, think those are leftover from when the code from test_framework/messages.py etc was copied in rather than imported